### DISCUSSION TITLE
Bridge direction

### DISCUSSION THREAD

**Discussion by sworcester001** [OrcaSlicer/OrcaSlicer#6354] [Q&A]
I have (essentially) a hollow toroid and when sliced, I can set bridge direction so it prints across the inside and outside, but it leaves two long straight bridges. Is there a way to have it bridges the inside to the outside "perpendicular" all the way around? Granted it would be little pie slices, but would be better bridging.
![bridge](https://github.com/user-attachments/assets/3f5972de-f2f2-47ab-b5c2-b857e621d5a9)

**Comment by BeeFox-sys** [comment]
did this ever get solved? i have a similar issue where i have a channel in the bottom of an object that is circular, meaning that i always have at least one second where the bridges are fairly long

**Reply by sworcester001** [reply]
No.
What I ended up doing was changing the bottom surface pattern to concentric and changing bridge flow ratio to .9 so it would be easier to make the bridge.
Being able to change the bridge patter would solve the problem though
